mne/viz/evoked.py

```diff
@@ -828,6 +844,23 @@ def plot_evoked_image(evoked, picks=None, exclude='bads', unit=True, show=True,
         resets the scale. Up and down arrows can be used to change the
         colormap. If 'interactive', translates to ``('RdBu_r', True)``.
         Defaults to ``'RdBu_r'``.
+    mask : ndarray | None
+        An array of booleans of the same shape as the data. Where this is
+        False, the resulting image is plotted transparently. Useful for,
```

Comment:
maybe: "Entries of the data that correspond to False in the mask are plotted transparently."

Reply:
Yeah makes sense.

Reply:
This is the opposite behavior of `imshow`, where if you plot a masked array, values where `mask==True` are masked.

Reply:
Like @mmagnuski , I think it's more important to be congruent with our cluster stats functions than with imshow.

Reply:
If this is consistent with the rest of MNE then fine with me (as long as it is clearly documented).